### DISCUSSION TITLE
驱除鞑虏，恢复中华

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#17] [Announcements]
# 驱除鞑虏，恢复中华

电影《满江红》本身有非常多值得探讨的点（贬义），但是宋词《满江红》本身是伟大的作品。片方近段时间来的一系列操作让我意识到，它配不上我写的影评，所以我被迫单拆一篇。

https://zexwoo.blog/posts/editorial/be-confident/

**Comment by daizc311** [comment]
是，但是。唉......

**Comment by ghost** [comment]
导演是不是说一开始《满江红》是要拍科幻片来着的哈哈哈挺好
